tuna/miopen/db/miopen_tables.py

```diff
@@ -337,52 +355,7 @@ class JobEnum(enum.Enum):
   evaluated_pend = 24
 
 
-class FinStep(enum.Enum):
-  """ Allowed Fin Steps """
-  # pylint: disable=invalid-name ; tuna/go_fish.py names valid fin steps as FinStep.__members__
-  find_compile = 1
-  find_eval = 2
-  get_solvers = 3
-  get_applicability = 4
-  not_fin = 5
-  miopen_find_compile = 6
-  miopen_find_eval = 7
-  miopen_perf_compile = 8
-  miopen_perf_eval = 9
-
-
-class JobMixin():
-  """Represents Mixin class for job tables"""
-
-  @declared_attr
-  def session(self):
-    """session key"""
-    return Column(Integer, ForeignKey("session.id"), nullable=False)
-
-  reason = Column(String(length=60), nullable=False, server_default="")
-  state = Column(Enum(JobEnum), nullable=False, server_default="new")
-  retries = Column(Integer, nullable=False, server_default="0")
-  result = Column(Text, nullable=True)
-
-  compile_start = Column(DateTime,
-                         nullable=False,
-                         server_default=sqla_func.now())
-  compile_end = Column(DateTime, nullable=False, server_default=sqla_func.now())
-  eval_start = Column(DateTime, nullable=False, server_default=sqla_func.now())
-  eval_end = Column(DateTime, nullable=False, server_default=sqla_func.now())
-
-  gpu_id = Column(Integer, nullable=False, server_default="-1")
-  machine_id = Column(Integer, nullable=False, server_default="-1")
-  eval_mid = Column(Integer, server_default="-1")
-  cache_loc = Column(Text)
-
-  solver = Column(String(length=128), nullable=True, server_default="")
-  fin_step = Column(mysql.MSSet(*(list(k for k in FinStep.__members__))),
-                    nullable=False,
-                    server_default="not_fin")
-
-
-class ConvolutionJob(BASE, JobMixin):
+class ConvolutionJob(BASE, JobMixin, MiopenJob):
```

Comment:
remove JobMixin from inheritance leave only MIOpenJob and please spell MIOpen not MiopenJob